test/core/channelz/channelz_test.cc

```diff
@@ -572,111 +572,6 @@ TEST_F(ChannelzRegistryBasedTest, GetTopChannelsMiddleUuidCheck) {
   }
 }
 
-TEST_F(ChannelzRegistryBasedTest, GetTopChannelsNoHitUuid) {
```

Comment:
Wouldn't we still want to run these tests if channelz_max_orphaned_nodes is 0?

Reply:
These tests were disabled for sharding (and we're removing non-sharding so they should be removed too).

They're built on assumptions that uuid generation happens at BaseNode creation time, and so they overly constrain the implementation in ways that we don't want.